matrix-sdk-android/src/main/java/org/matrix/android/sdk/internal/util/UrlUtils.kt

```diff
@@ -39,7 +39,7 @@ internal fun String.ensureProtocol(): String {
 }
 
 /**
- * Ensure string has trailing /
+ * Ensure string has trailing /.
```

Comment:
NIT, but at first I thought that this sentence is just cut off by mistake. It would be cool to take / in quotes to make it clear, but I'm not sure how to escape a single quote in kDoc. Possible option is Html escape like 

&lsquo / &lsquo

but it looks ugly when you read doc in file, not by viewer. Maybe taking in double quotes inside markdown block will be better choice, like this

\`"/"\`

Reply:
Clarified in dcbc85484c2f321367244ece4e480296bed436d3